tests/unit/local/docker/test_container.py

```diff
@@ -567,17 +615,17 @@ def test_wait_for_result_error_retried(self, mock_requests):
             calls,
             [
                 call(
-                    "http://localhost:7077/2015-03-31/functions/function/invocations",
+                    "http://None:7077/2015-03-31/functions/function/invocations",
```

Comment:
What is this testing?

Reply:
It's to test whether the URL can be replaced by the new option. None is the default value of container_host in container class. So if we don't assign the value here, the URL should be replaced to this.

Reply:
I see. I'd probably pass `localhost` explicitly then as a principle of least surprise (seeing `None` in an URL is a little weird).

Reply:
Setting `localhost` here is fine. The only thing I worry about is that it will override the previous unit test (the URL will only be replaced when setting `container_host`). Is it okay?
If it's okay, I will remove the new unit test(using `host.docker.internal` as container_host) as well since it basically does the same thing.

Reply:
If they're both testing the same thing, then sure there's no point in having two tests.